google/cloud/forseti/services/inventory/storage.py

```diff
@@ -110,8 +111,10 @@ def create(cls):
             object: InventoryIndex row object.
         """
 
+        start_time = cls._utcnow()
         return InventoryIndex(
-            start_time=cls._utcnow(),
+            id=start_time.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-1],
```

Comment:
I see this is to address PR #992.  But, do you think that id would be unique among multiple potential forseti instances that a customer may have?  What if they run on a cron job, that triggers at the same time, e.g. on the hour?

Reply:
I discussed with Andrew this morning. We think changing id to use timestamp can solve the issue of collision and it is easy to sort. The stamp generated here is like:'2018-01-30T04:30:34.16326'. 2 digits more than ms. It will be very very unlikely to cause collision using cron job.
In addition to that. My opinion is that a new column for root id is unnecessary since it can be acquired without that column.